staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -211,6 +211,14 @@ type ObjectMetricSource struct {
 	MetricName string `json:"metricName" protobuf:"bytes,2,name=metricName"`
 	// targetValue is the target value of the metric (as a quantity).
 	TargetValue resource.Quantity `json:"targetValue" protobuf:"bytes,3,name=targetValue"`
+
+	// selector is the string-encoded form of a standard kubernetes label selector for the given metric
```

Comment:
Explain what selector means to the user in this context.  Why would they set it, what happens if it is empty, etc.

Reply:
@smarterclayton updated with your feedback, is there anything else that stands out?